src/style.css

```diff
@@ -0,0 +1,16 @@
+@import url(./blocks/card.css);
```

Comment:
it seems like your styles don't apply
![image](https://user-images.githubusercontent.com/86923089/213913854-3c6c7759-7724-4263-b136-746d6c8f6ec4.png)

fix it, if you need help - ask in chat